.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Feature/Enhancement request
+about: Suggest an idea for this project
+title: "[ENH]"
+labels: enhancement
+assignees: ''
+---
+
+**Is your feature request related to a problem/struggle? Please describe.**
+A clear and concise description of what the problem is your idea tries to sovle. E.g: I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
+
+**Additional context**
```

Comment:
As a header? (`#Additional context`)

Reply:
Apply headings to all questions, or just 'Additional context'? If so could you elaborate why? The bug report format doesn't have a heading for it's additional context either.